### PR TITLE
Update tg.js (Tagalog Language based on 4.0.9)

### PR DESCRIPTION
**Pull request by Navsy17**
Adding changes based from the new v4.0.9 English Build. Looking for additional helpers, Help appreciated.

### CODE REVIEW

**Comment by mty22** [comment]
Looks good! pending review from native speaker.

**Comment by Navsy17** [comment]
Need help for reviewer

**Comment by Navsy17** [comment]
Dandan and Dreamcatcher offered to help in review, 

Main objective: find shorter and much more suitable alternatives, (turns out a lot of the words don't fit and are too long)
Plus need help with some grammar issues, thx

**Comment by Dreamcatcher7D** [comment]
In "dye sa katawan"
We Could do 'Kulay ng damit' or 'Tinay ng katawan', both sound smoother and easier to underand.

File: tg.js [hunk 1]
```diff
@@ -303,10 +303,10 @@ settings.audio.asset Bolyum ng Asset
 settings.audio.dialogue Bolyum ng dialogo
 settings.viewmodel.header Tingnan ang Modelo
 settings.viewmodel.vmSetts Settings ng View Model
-settings.viewmodel.fov Field of View (Larangan ng Pagtingin)
+settings.viewmodel.fov Field of View (Sakop ng nakikita)
 settings.viewmodel.fps FOV ng Armas
 settings.viewmodel.bobbing Pagtalbog ng Armas
-settings.viewmodel.leaning Pagkasandal ang Armas
+settings.viewmodel.leaning Pagsandal ang Armas
```
Comment (Dreamcatcher7D) [inline tg.js:309]: Instead of 'ang' we can do 'ng', grammatically correct, and smoother.

File: tg.js [hunk 2]
```diff
@@ -781,7 +781,7 @@ windows.clan.leave Sigurado ka bang nais mong umalis sa hukbo/samahan ninyo?
 windows.clan.delete Sigurado ka bang nais mong tanggalin ang hukbo/samahan?
 windows.clan.kick Sigurado ka bang nais mong i-kick si {0} at tanggalin sa samahan?
 windows.store.premium.purchase Sigurado ka bang nais mong bumili ng {0} araw ng Premium?
-windows.store.premium.gift Are you sure you want to Gift '{0}' {1} days of Premium?
+windows.store.premium.gift Sigurado nais mo bang regaluhin si '{0}' ng {1} na araw ng Premium?
 windows.playerlist.ban Sigurado ka bang nais mong ipagbawal si {0} mula sa larong ito?
 windows.playerlist.flag Sigurado ka bang nais mong I-flag si {0}?
 windows.playerlist.unflag Sigurado ka bang nais mong mag-Unflag si {0}?
```
Comment (Dreamcatcher7D) [inline tg.js:787]: Instead of 'mag-Unflag', it should be 'I-Unflag'

File: tg.js [hunk 3]
```diff
@@ -98,7 +98,7 @@ settings.experimental.header Experimental na Header
 settings.experimental.aimFreezeFix Ayusin ang Aim Freeze
 settings.experimental.mouseFlickFix Pag-ayos ng Mouse Flick
 settings.experimental.shaderRendering Pag-render
-settings.experimental.staticRendering Static Rendering
+settings.experimental.staticRendering Static na Pag-render
```
Comment (Dreamcatcher7D) [inline tg.js:101]: We can do 'Static na Rendering'.

File: tg.js [hunk 4]
```diff
@@ -262,7 +262,7 @@ settings.gameprivacy.streamer Mode ng Streamer
 settings.gameprivacy.anon Anonymous Mode
 settings.gameprivacy.hideGameTitle Itago ang Pamagat ng Laro
 settings.gameplay.header Gameplay
-settings.gameplay.gpSetts Gameplay Settings
+settings.gameplay.gpSetts Settings sa Gameplay
```
Comment (Dreamcatcher7D) [inline tg.js:265]: We could use 'ng' instead of 'sa'

**Review by Dreamcatcher7D — DISMISSED** [review]

File: tg.js (hunk 3 above)
Comment (Navsy17) [reply tg.js:101]: will do, thanks

**Comment by mty22** [comment]
If @Navsy17 has finished with updates, @Dreamcatcher7D please review let me know when good to merge. :)

File: tg.js [hunk 5]
```diff
@@ -57,12 +57,12 @@ generic.username Username
...
 generic.register Mag-parehistro
 generic.login Mag log-in
-generic.accuracy Kawastuhan ng pagbaril
+generic.accuracy Accuracy sa paglalaro
 player.waiting Naghihintay ng mga manlalaro
 player.died Ikaw ay namatay
-player.killed {0} Pumatay kay {1}
-player.killstreak {0} nasa {1} Kill Streak
-player.inactive Pina-alis dahil sa walang aktibidad
+player.killed si {0} ay pumatay kay {1}
```
Comment (callmedandann) [inline tg.js:63]: Instead of "Si Player A ay pumatay kay Player B", make it "Pinatay ni Player A si Player B" because its a bit long

File: tg.js [hunk 6]
```diff
@@ -72,19 +72,19 @@ app.spectate Pindutin upang manood
...
+settings.reset.confirm Sigurado ka bang nais mong I-reset ang inyong setitngs?
 settings.import angkat
 settings.export iluwas
-settings.none Walang nahanap na mga settings
+settings.none Walang nahanap na mga settings.
 settings.backup Mga back-up settings
 settings.controls.change Lokal na Rehiyon 
 settings.controls.press Pindutin ang anumang key
-settings.local.header Localization <span id="requiresRestart"><span style="color: #eb5656">*</span> nangangailangan ng restart</span>
+settings.local.header LoKalisasyon <span id="requiresRestart"><span style="color: #eb5656">*</span> nangangailangan ng restart</span>
```
Comment (callmedandann) [inline tg.js:87]: Make the K from Lokalisasyon small

File: tg.js [hunk 7]
```diff
@@ -57,12 +57,12 @@ generic.username Username
 generic.password  Illagay ang password
 generic.register Mag-parehistro
 generic.login Mag log-in
```
Comment (callmedandann) [inline tg.js:59]: Could it be 'Mag Log-in'?

**Comment by StarRayX** [comment]
Hello, I'm here to help again

File: tg.js [hunk 8]
```diff
@@ -1,821 +1,821 @@
...
+windows.publish.map.login Lumikha at I-upload ang laro!
+windows.theatre.header Modo na Theatre
+windows.theatre.record Paganahin ang Pagre-record
+windows.theatre.key Pang-record
+windows.clans.header Samahan
+windows.clans.members Mga Miyembro ({0})
+windows.clans.requests Mga Nais Sumali 
+windows.clans.requests.none Walang Nais Sumali
+windows.clans.page [{0}] Pahina ng Samahan
+windows.clans.create Lumikha ng Samahan
```
Comment (StarRayX) [inline tg.js:527]:
```suggestion
windows.clans.create Gumawa ng Samahan

```

File: tg.js [hunk 9]
```diff
@@ -1,821 +1,821 @@
...
+windows.clans.name Pangalan ng Samahan
+windows.clans.view Tignan ang mga Samahan
+windows.clans.join Sumali sa Samahan
+windows.clans.login Mag-login upang makagawa o makasali sa isang Samahan!
+windows.store.header Tindahan
+windows.store.spin {0} Premyo
+windows.store.agree Sa pamamagitan ng pagbili sumasang-ayon ka sa {0} Mga Tuntunin
+windows.store.purchase Bumili ng KR
+windows.store.voucher Kunin ang voucher
```
Comment (StarRayX) [inline tg.js:536]:
```suggestion
windows.store.voucher Gamitin ang voucher
```
Using kunin in this context would translate to "take the voucher"

File: tg.js [hunk 10]
```diff
@@ -1,821 +1,821 @@
...
+windows.store.voucherinfo Kumuha ng mga Voucher mula sa maraming mga Streamer at kaganapan
+windows.store.market Palengke
```
Comment (StarRayX) [inline tg.js:538]:
```suggestion
windows.store.market Merkado
```
For uniformity

File: tg.js [hunk 11]
```diff
@@ -1,821 +1,821 @@
...
+windows.store.market2 Krunker Merkado
+windows.store.premium Bumili ng Premium
+windows.store.premium2 Krunker Premium
+windows.store.buyP Pumili ng isang Display name
+windows.store.buy Bumili at Magbenta ng mga kagamitan
+windows.store.wheels Mga Prize Wheels
+windows.store.unlock I-unlock ang mga bagong disenyo ng mga armas at kagamitan
+windows.store.amount {0} KR sa account
+windows.store.login upang bumili at mag-unlock ng mga item mula sa tindahan!
+windows.store.purchasing Bumibili ...
```
Comment (StarRayX) [inline tg.js:548]:
```suggestion
windows.store.purchasing Binibili ...

```

File: tg.js [hunk 12]
```diff
@@ -1,821 +1,821 @@
...
+settings.gameprivacy.streamer Mode ng Streamer
+settings.gameprivacy.anon Anonymous Mode
+settings.gameprivacy.hideGameTitle Itago ang Pamagat ng Laro
+settings.gameplay.header Gameplay
+settings.gameplay.gpSetts Settings sa Gameplay
+settings.gameplay.sensitivityX X Sensitivity
+settings.gameplay.sensitivityY Y Sensitivity
+settings.gameplay.aimsensX  X Sensitivity sa pagpuntirya (Aim)
+settings.gameplay.aimsensY  Y Sensitivity sa pagpuntirya (Aim)
+settings.gameplay.challenge Mga Pagsubok at Hamon
```
Comment (StarRayX) [inline tg.js:270]:
```suggestion
settings.gameplay.challenge Challenge Mode
```

File: tg.js [hunk 13]
```diff
@@ -1,821 +1,821 @@
...
+windows.client.windows Installer sa Windows
+windows.client.mac Installer sa MacOS
+windows.client.linux Installer sa Linux
+windows.ranked.header Rango
+windows.ranked.login Login upang magpataasan ng inyong rango!
+windows.ranked.level Hindi ka bababa sa Antas na <span color = "black"> level {0} </span> upang maglaro sa may ranggo. Ikaw ay kasalukuyang nasa Antas <span color = "black"> level {1} ​​</span>.
+windows.ranked.mode.r1v1 Solo
+windows.ranked.mode.r2v2 Duo
+windows.ranked.mode.r4v4 Quad
+windows.chall.header Mga Hamon
```
Comment (StarRayX) [inline tg.js:600]:
```suggestion
windows.chall.header Mga Challenge

```

File: tg.js [hunk 14]
```diff
@@ -1,821 +1,821 @@
...
+windows.chall.login upang makumpleto ang mga hamon!
```
Comment (StarRayX) [inline tg.js:601]:
```suggestion
windows.chall.login upang makumpleto ang mga challenge!

```

**Review by StarRayX — CHANGES_REQUESTED** [review]

**Review by StarRayX — CHANGES_REQUESTED** [review]

File: tg.js [hunk 15]
```diff
@@ -1,821 +1,821 @@
...
+social.trades.out Palabas
+social.trades.out.head Palabas na Pag-aalok ({0})
+social.trades.history Kasaysayan
+social.trades.history.head Kasaysayan ng Pag-kakalakalan ({0})
+social.trades.decline Tanggihan Lahat
+social.trades.message Payagan ang mga Trade Message at Pag-alok sa Settings
+social.trades.warning Wala kang makukuhang kagamitan sa Pag-alok na ito!
+social.trades.accept Nais mong tanggapin ang Pag-alok na ito?
+stats.airdrops KR-Packages
+settings.quality.flapAnim Animasyon ng Sniperflap
```
Comment (callmedandann) [inline tg.js:820]: Can you add space between Sniper And flap?

Comment (StarRayX) [reply tg.js:820]: you can request a change

File: tg.js [hunk 16]
```diff
@@ -1,821 +1,821 @@
...
+settings.client.unlimitedFrames Unlimited na FPS
+settings.client.gameCap Record ng Laro
+settings.client.experimental Experimental Features
+settings.client.discordRPC RPC sa Discord
+settings.client.resourceSwapper Pagpalit ng Resources
+settings.client.clearCache I-clear ang Cache
+settings.client.resetSettings I-reset ang Settings ng Kliyent
+settings.client.acceleratedCanvas Accelerated ang Canvas
+settings.client.angleBackend ANGLE Graphics ang Backend
+settings.client.colorProfile Color ng Profile
```
Comment (StarRayX) [inline tg.js:112]:
```suggestion
settings.client.colorProfile Profile ng Kulay
```
Current translation restranslates to "Profile color" and not "Color Profile"

**Review by StarRayX — CHANGES_REQUESTED** [review]

File: tg.js (hunk 5 above)
Comment (Navsy17) [reply tg.js]: I committed it, with the other suggestions of yours, Thanks.

File: tg.js (hunk 7 above)
Comment (Navsy17) [reply tg.js]: I committed it, with the other suggestions of yours, Thanks, did some changes to make it shorter.

File: tg.js (hunk 8 above)
Comment (Navsy17) [reply tg.js:527]: Lumikha also means, "create". Thanks for considering this option, will think about it.

File: tg.js [hunk 17]
```diff
@@ -1,821 +1,821 @@
...
+windows.chall.header Mga Hamon
+windows.chall.login upang makumpleto ang mga hamon!
+thumbnail.upload.limit Failed. {0}kb Thumbnail Limit
+image.upload.limit Failed. {0}kb ang Limitasyon ng Larawan
+thumbnail.upload.error Thumbnail Error
+mod.extracting Kumukuha ng {0} / {1}
+mod.disabled hindi pinagana ang mga mod
+mod.error Error sa mod!
+mod.loading Naglo-load ng mod ...
+mod.invalid Di-wastong mod file!
```
[the remark this answers is not in the document]
Comment (Navsy17) [reply tg.js:609]: Interesting option. Will add.

File: tg.js [hunk 18]
```diff
@@ -1,821 +1,821 @@
...
+stats.thrownkills Mga Nabatuhan ng Patalim
+stats.headshots Mga Headshot
+stats.wallbangs Mga Wallbang
+stats.sprays Nilagay na spray
+windows.host.default.settings Ang pagpapagana ng Default na Mga Setting ay nagbibigay-daan para sa pagtatago ng Player Progress at Highscores
+windows.host.default.commands * Ang Mga Command ng Host ay Hindi Pinapagana sa mode na ito: / noclip / god / smite atbp
+windows.host.rewards Pinipilit ng pagpapagana ng KR Rewards ang Server na gumamit ng Default na Mga Setting
+windows.store.wheels.sub Subukan ang iyong kapalaran at paikutin ang Prize wheel para sa isang pagkakataon upang manalo ng ilang mga bagong disenyo!
+windows.store.currency * Ang mga presyo sa USD ay maaaring mai-convert sa iyong lokal na pera sa isang rate ng palitan na tinukoy ng provider ng pagbabayad o bangko
+windows.store.creator Tangkilikin ang inyong paboritong Content Creator o Streamer sa pamamagitan ng 'Creator Code'!
```
[the remark this answers is not in the document]
Comment (Navsy17) [reply tg.js:776]: Perhaps, though most of my peers do know this. Speaking from the perspective in where I live.

File: tg.js [hunk 19]
```diff
@@ -1,821 +1,821 @@
...
+social.profile.mods.none Walang akdang Mods mula sa User
+social.search Resulta ng Paghahanap
+social.market.stats.rare Bihirang Mahanap na Items
+social.market.stats.circulation Nasa Sirkulasyon
+social.market.stats.units Bilang ng item na ito pandaigdig
+social.trades.in Papasok
+social.trades.in.head Pumapasok na Pag-aalok ({0})
+social.trades.out Palabas
+social.trades.out.head Palabas na Pag-aalok ({0})
+social.trades.history Kasaysayan
```
[the remark this answers is not in the document]
Comment (Navsy17) [reply tg.js:813]: Tentative option.

File: tg.js [hunk 20]
```diff
@@ -1,821 +1,821 @@
...
+social.trades.history.head Kasaysayan ng Pag-kakalakalan ({0})
+social.trades.decline Tanggihan Lahat
+social.trades.message Payagan ang mga Trade Message at Pag-alok sa Settings
+social.trades.warning Wala kang makukuhang kagamitan sa Pag-alok na ito!
+social.trades.accept Nais mong tanggapin ang Pag-alok na ito?
+stats.airdrops KR-Packages
```
[the remark this answers is not in the document]
Comment (Navsy17) [reply tg.js:819]: Thanks for this one, had my thoughts rolling on what to use

File: tg.js [hunk 21]
```diff
@@ -1,821 +1,829 @@
...
+custom.host.ranked Hindi ma-host habang naglalaro sa 'Ranggo'
+funds.insufficient Hindi sapat na KR
+funds.out Ubos ang KR
+custom.passcode.long Masyadong mahaba ang password 
+custom.passcode.input Ipasok ang password ng server upang sumali
+custom.tryagain subukang muli
+purchase.kr.success Idinagdag ang {0} KR sa account mo
+limited.purchase.ask Sigurado ka bang bumili ng  [{0}] para sa{1} KR?
+limited.purchase.success Idinagdag ang sa inventory mo
+pretend.exit pindutin muli ang[{0}] again to exit
```
Comment (StarRayX) [inline tg.js:754]:
```suggestion
pretend.exit Pindutin muli ang [{0}] to exit
```

File: tg.js [hunk 22]
```diff
@@ -1,821 +1,829 @@
...
+respawn.auto Auto Respawning sa {0}
+custom.host.ranked Hindi ma-host habang naglalaro sa 'Ranggo'
+funds.insufficient Hindi sapat na KR
+funds.out Ubos ang KR
+custom.passcode.long Masyadong mahaba ang password 
+custom.passcode.input Ipasok ang password ng server upang sumali
+custom.tryagain subukang muli
+purchase.kr.success Idinagdag ang {0} KR sa account mo
+limited.purchase.ask Sigurado ka bang bumili ng  [{0}] para sa{1} KR?
+limited.purchase.success Idinagdag ang sa inventory mo
```
Comment (StarRayX) [inline tg.js:753]:
```suggestion
limited.purchase.success Idinagdag ang item sa iyong inventory
```

File: tg.js [hunk 23]
```diff
@@ -1,821 +1,829 @@
...
+generic.none Wala
+generic.connecting KUMOKONEKTA...
+generic.select Pumili
+generic.delete Tanggalin
+generic.save Mag-save
+generic.load Mag-Load
+generic.search Humanap
+generic.leave Umalis
+generic.default Default
+generic.submit Sumite
```
Comment (StarRayX) [inline tg.js:25]:
```suggestion
generic.submit I-Sumite
```

File: tg.js [hunk 24]
```diff
@@ -1,821 +1,829 @@
...
+social.market.shoe Mga Sapatos 
+social.market.pet Mga Alaga
+social.market.melee Mga Patalim
+social.market.spray Mga Sprays
+social.market.dye Kulay ng Katawan
+social.market.none Hindi matagpuan ang kagamitan
+social.market.login Maglogin ka muna.
+social.market.grace handa na sa {0}
+social.market.sort Pag-uri-uriin
+social.market.filter I-Filter sa Paghahanap
```
Comment (StarRayX) [inline tg.js:736]:
```suggestion
social.market.filter I-Filter ang Paghahanap
```

File: tg.js [hunk 25]
```diff
@@ -1,821 +1,829 @@
...
+social.market.sort2 Pagbukud-bukurin
+social.market.price.high Pinakamataas na Presyo
+social.trading.submitting Nagsusumite ng Alok ...
```
Comment (StarRayX) [inline tg.js:739]:
```suggestion
social.trading.submitting Pinapadala ang Alok ...
```

**Review by StarRayX — CHANGES_REQUESTED** [review]

**Comment by mty22** [comment]
@Navsy17 let me know when good to go~

**Comment by Navsy17** [comment]
Fixed the merge conflict. can now merge

File: tg.js (hunk 15 above)
[the remark this answers is not in the document]
Comment (Navsy17) [reply tg.js]: Done, thanks.

Comment (Navsy17) [reply tg.js]: Done, committed.

File: tg.js (hunk 18 above)
[the remark this answers is not in the document]
Comment (Navsy17) [reply tg.js]: Personally, I think 'tangkilikin' fits more. It is more direct to patronizing.

File: tg.js (hunk 3 above)
Comment (Navsy17) [reply tg.js]: Already committed with the others. Thanks.

File: tg.js (hunk 8 above)
Comment (Navsy17) [reply tg.js]: Lumikha seems more formalized. I will still consider this in the future, Thanks for the note.

File: tg.js (hunk 19 above)
[the remark this answers is not in the document]
Comment (Navsy17) [reply tg.js]: Decided to do Talaan which can fit both the roles of kasaysayan and historya.

**Comment by Navsy17** [comment]
@mty22  Tagalog for v4.0.9 is now good to go.

**Review by mty22 — APPROVED** [review]

**Review by callmedandann — APPROVED** [review]

**Comment by mty22** [comment]
@callmedandann and @Dreamcatcher7D  can you guys pls send me your IGNs?

**Comment by mty22** [comment]
kr queued

**Pull request merged**